extensions/wikia/AdEngine/js/lookup/prebid/adapters/beachfront.js

```diff
@@ -0,0 +1,65 @@
+/*global define*/
+define('ext.wikia.adEngine.lookup.prebid.adapters.beachfront', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.context.slotsContext',
+	'ext.wikia.aRecoveryEngine.instartLogic.recovery',
+	'wikia.log'
+], function (adContext, slotsContext, instartLogic, log) {
+	'use strict';
+
+	var bidderName = 'beachfront',
+		bidfloor = 0.01,
+		logGroup = 'ext.wikia.adEngine.lookup.prebid.adapters.beachfront',
+		slots = {
+			oasis: {
+				INCONTENT_PLAYER: {
+					exchangeId: 'd239e601-dd57-4163-fe5d-35012d77395f'
+				}
+			},
+			mercury: {
+				MOBILE_IN_CONTENT: {
+					exchangeId: 'f377a8b1-c5c0-4108-f932-0102a81ff43d'
```

Comment:
Can we use name suggested by prebid docs for this string? It makes code harder to follow when we have multiple names for the same id.